go-backend/db/recognition_hi5.go

```diff
@@ -0,0 +1,78 @@
+package db
+
+import (
+	"context"
+	logger "github.com/sirupsen/logrus"
+	"time"
+)
+
+const (
+	createRecognitionHi5Query = `INSERT INTO recognition_hi5 (
+		recognition_id,
+		comment,
+		given_by,
+		given_at
+		) VALUES ($1, $2, $3, $4);`
+
+	updateHi5QuotaQuery = `UPDATE users SET hi5_quota_balance = $1 where id = $2;`
+)
+
+type RecognitionHi5 struct {
+	Id				int			`db:"id" json:"id"`
+	RecognitionId			int			`db:"recognition_id" json:"recognition_id"`
```

Comment:
when using abbreviations, every thing should be caps
so `RecognitionID` instead of `RecognitionId`